learn-beta2.md

```diff
@@ -42,10 +30,14 @@ A series of guides on various things you need to do during the Ballerina develop
 
 ## References
 
-### [Language Introduction Slides](http://localhost:4000/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)
+### [Language Introduction Slides](/learn/language-concepts/Ballerina_Swan_Lake_Presentation_Deck_V1.0.pdf)
 
 Presentation slides on the Ballerina language that anyone can use to talk about the language.
 
+### [Language Walkthrough Slides](/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)
```

Comment:
Language Walkthourgh video is having under `Getting Started`. It seems this is the slide set used for that video. So shouldn't we have that slides next to the video. Otherwise having the two forms of the same thing in two different sections doesn't look ok to me.

Reply:
Updated.